src/frontend/src/lib/stores/i18n.store.ts

```diff
@@ -3,10 +3,10 @@ import en from '$lib/i18n/en.json';
 import { getLocalStorageLang, setLocalStorageItem } from '$lib/utils/local-storage.utils';
 import { writable, type Readable } from 'svelte/store';
 
-const zhCnI18n = async (): Promise<I18n> => {
+const loadI18nFile = async (lang: Languages): Promise<I18n> => {
```

Comment:
These changes in the `i18n.store` are incorrect but, I'll go on and fix it.

Reply:
There https://github.com/buildwithjuno/juno/commit/6152af5699d8fe42e84fda32e500351aefefd319

It was a great idea but the lazy loaded import needed a static path. All good now 😃

Reply:
Ah okok perfect then.
I just did it in good faith because I thought it was a good way for when other languages will come in the future, because there would not had the need to add more code in that file for each new language :)
thanks for merging the i18n btw, now Juno can speak italian! :)